mycore-iiif/src/main/java/org/mycore/iiif/presentation/resources/MCRIIIFPresentationResource.java

```diff
@@ -0,0 +1,173 @@
+/*
+ *  This file is part of ***  M y C o R e  ***
+ *  See http://www.mycore.de/ for details.
+ *
+ *  This program is free software; you can use it, redistribute it
+ *  and / or modify it under the terms of the GNU General Public License
+ *  (GPL) as published by the Free Software Foundation; either version 2
+ *  of the License or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful, but
+ *  WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program, in a file called gpl.txt or license.txt.
+ *  If not, write to the Free Software Foundation Inc.,
+ *  59 Temple Place - Suite 330, Boston, MA  02111-1307 USA
+ *
+ */
+
+package org.mycore.iiif.presentation.resources;
+
+import java.util.Date;
+import java.util.concurrent.TimeUnit;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Response;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.mycore.common.MCRCache;
+import org.mycore.iiif.presentation.MCRIIIFPresentationManifestQuickAccess;
+import org.mycore.iiif.presentation.impl.MCRIIIFPresentationImpl;
+import org.mycore.iiif.presentation.model.basic.MCRIIIFManifest;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+import static org.mycore.iiif.presentation.MCRIIIFPresentationUtil.correctIDs;
+
+@Path("/iiif/presentation")
+public class MCRIIIFPresentationResource {
+
+    private static final Logger LOGGER = LogManager.getLogger();
+
+    private static final MCRCache<String, MCRIIIFPresentationManifestQuickAccess> cache = new MCRCache<>(1000,
+        MCRIIIFPresentationResource.class.getName().toString());
+
+    private static final String APPLICATION_LD_JSON = "application/ld+json";
+
+    private static final String IMPL = "impl";
+
+    /* currently not supported by MyCoRe
+        @GET
+        @Produces(APPLICATION_LD_JSON)
+        @Path("{impl}/collection/{name}")
+        public Response getCollection(@PathParam(IMPL) String impl, @PathParam("name") String name) {
+            return null;
+        }
+    */
+
+    @GET
+    @Produces(APPLICATION_LD_JSON)
+    @Path("{impl}/{identifier}/manifest")
```

Comment:
every method starts with `impl` path parameter. Should be path parameter of class